### PR TITLE
Another take at being very explicit about permissions

### PR DESCRIPTION
**Pull request by ZachGoldberg**
Also ran this by Brian, his feedback is that this is clear/concise for onboarding customers

### CODE REVIEW

**Comment by ZachGoldberg** [comment]
@yhakbar thoughts?

File: docs/2.0/docs/pipelines/installation/viagithubapp.md [hunk 1]
```diff
@@ -35,15 +35,13 @@ As of 2024/09/10, the Gruntwork.io GitHub App requests the following permissions
...
 
-  1. You are using the app to create extra repositories during initial bootstrap. By default, the [infrastructure-live-root-template](https://github.com/gruntwork-io/infrastructure-live-root-template) repository is used to vend a single repository (the `infrastructure-live-root` repository), but the workflow for bootstrapping it can also be used to create additional optional repositories (`infrastructure-live-access-control` and `infrastructure-modules`).
-  2. You are using the app to vend new repositories via Account Factory. This is the case for DevOps Foundations Enterprise customers using Account Factory delegated repository vending.
-  3. You are using the app to manage access to repositories. This is the case for DevOps Foundations Enterprise customers using team management configurations for Account Factory delegated repositories.
+The reasoning for this is that if you are using Account Factory to create delegated repositories then you'll need to either allow the app to have access to the entire organization or manually intervene and update the apps configuration on every new account vend.
```
Comment (yhakbar) [inline docs/2.0/docs/pipelines/installation/viagithubapp.md:43]: That's not even possible, as the Pipeline updates the repo immediately after vending it. Enterprise customers don't have a choice here.

**Review by yhakbar — CHANGES_REQUESTED** [review]
One small change request, but otherwise LGTM.

File: docs/2.0/docs/pipelines/installation/viagithubapp.md (hunk 1 above)
Comment (ZachGoldberg) [reply docs/2.0/docs/pipelines/installation/viagithubapp.md]: Good callout, updated!  Thank you

**Review by yhakbar — APPROVED** [review]

**Pull request merged**